docs/outlook/turn-exchange-tokens-on-off.md

```diff
@@ -64,8 +64,10 @@ Blocked: []
 PS C:\>
 ```
 
+If any add-ins requested a legacy Exchange Online token in the last seven days, it will appear in the **Allowed** or **Blocked** list. If an add-in was granted the token request, it appears in the **Allowed** list. If the token request was denied, it appears in the **Blocked** list. It is possible for the same add-in to appear in both lists. This can happen if they were allowed to get tokens, but then tokens were turned off.
```

Comment:
```suggestion
If any add-ins requested a legacy Exchange Online token in the last seven days, it will appear in the **Allowed** or **Blocked** list. If an add-in was granted the token request, it appears in the **Allowed** list. If the token request was denied, it appears in the **Blocked** list. It's possible for the same add-in to appear in both lists. This can happen if they were allowed to get tokens, but then tokens were turned off.
```